app/models/activity.rb

```diff
@@ -82,6 +82,16 @@ class Activity < ApplicationRecord
                                    .confirmed.by_dates(period_begin, period_end)
                                    .order(occur_at: 'desc')
                                }
+  def accompaniable?
+    event != 'family_court' || accompaniments.count < 3
```

Comment:
We should do volunteer_accompaniments.count rather than accompaniments.count because the cap only applies to users with the volunteer role who are accompanying.  Users with the accompaniment_leader role do not factor into the cap.